spec/models/schedule_generator_service_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe ScheduleGenerator, type: :model do
+  describe '#generate_schedule' do
+    subject(:generate_schedule) { ScheduleGenerator.new(current_member).generate_schedule }
+
+    let(:current_member) { create :member }
```

Comment:
I think it would be nicer to have this test variable called `author` instead of `current_member`, as it would state explicitly the role the the member passed to the `ScheduleGenerator`: it is used to set the `author` attribute of the created Missions